tests/smoketests/test_coalescent.py

```diff
@@ -0,0 +1,103 @@
+import dendropy
```

Comment:
There's a tool called `isort` that will alphabetize imports and arrange them nicely; you can install it with pip.  Let's run it on this file `python3 -m isort test_coalescent.py`

Reply:
Ah, also let's run the black formatter on this file if you haven't already.